UI/tests/common/mocks/lsmb_elements.js

```
@@ -1,4 +1,7 @@
 /** @format */
+/* eslint-disable vue/component-definition-name-casing */
+/* eslint-disable vue/require-prop-types, vue/one-component-per-file */
```

require-prop-types is globally disabled, no?

---

Yes, for vue files

---

What do you mean by that? Why does this file want to report a vue error in the first place? It's not a Vue file? (Can't we disable Vue checks for non-Vue files?)

---

`require-prop-types` is only globally disabled in the vue section, not in the js one. `lsmb_elements.js`  is a vue component